CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Smart Search - Limit search to specific collection fields.
```

Comment:
That is not the only thing you did in this PR (you should talk about the new Smart Actions "API").